FinSearchUnified/Controllers/Frontend/Findologic.php

```diff
@@ -9,13 +9,17 @@ public function indexAction()
         $shopKey = $this->request->get('shopkey');
         $start = (int)$this->request->getParam('start', 0);
         $count = (int)$this->request->get('count');
+        $productId = (int)$this->request->get('productId');
```

Comment:
You may not want to cast this into an `int` for example a manufacturerId may also have some characters, which are not just numbers.